OpenNMS-Cloud/horizon-stream
# HS-848 GraphQL endpoints include the node scan results

## Description

How to test:
- Enable SNMP in the local environment (laptop)
- Build a local minion docker images
- Follow the instruction on the page https://github.com/OpenNMS-Cloud/horizon-stream/wiki/Getting-Started---Admin to create an external local minion. 
- Add the local device (laptop) as a new node with the external minion location. 
- Verify the node info, newly detected IpInterfaces, and SNMP interfaces are populated with GraphQL query at http://localhost:8123/api/gui

## Jira link(s)
- https://issues.opennms.org/browse/HS-848

## Flagged for review


## Checklist
* [x] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [x] Appropriate reviewer(s) have been selected.
* [x] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
